src/connection.ts

```diff
@@ -91,21 +91,25 @@ export class WebRTCConnection implements ic.Connection {
 
       log.trace(`${logPrefix} supported protocols - ${protocols}`);
```

Comment:
out of scope for this PR but why do we not inject the formatting logic (e.g. `logPrefix`) as part of our logging setup/functions? Seems a bit silly that we need to add it for each line here.

Reply:
Yeah, this is an artifact of debugging.